specs/angular-readonly-form-styling/README.md

```diff
@@ -0,0 +1,118 @@
+# Angular read-only form styling
+
+## Problem Statement
+
+There is a desire to render Angular forms with read-only controls rather than disabled controls when the control in the Angular form is marked as disabled. This is because there are use cases where the configuration represented by nimble controls is not editable but is still relevant and should be easily readable. The only way to mark something in an Angular form as "not editable" is to mark it as disabled because there is no concept of "read only" in an Angular form. There is quite a bit of logic built into Angular forms and Angular's control value accessors around what it means to render a component bound to a disabled form control.
+
+With some changes to nimble's controls, applications could much more easily create forms with read-only controls rather than disabled controls.
+
+An example can be seen below of a page that currently has disabled controls and the ideal UI.
+
+Current UI with disabled controls:
+
+![disabled form](disabled-form.png)
+
+Improved UI with read-only controls:
+![readonly form](readonly-form.png)
+
+Note that in the improved UI screenshot, there are likely still additional improvements that should be made. These improvements include, but are not limited to, updating the `nimble-switch` and `nimble-checkbox` instances to no longer be disabled. However, a decision has not yet been made regarding how these components should be represented when read-only.
+
+## Links To Relevant Work Items and Reference Material
+
+[SLE control pattern docs](https://stratus-storybook.ni.dev/?path=/docs/patterns-control-text--docs)
+
+## Implementation / Design
+
+To enable this new design, the following changes need to be made within the nimble repo, each of which is discussed in more detail below:
+
+- Introduce `readonly-disabled-appearance` configuration on the `nimble-text-field`, `nimble-number-field`, `nimble-text-area`, `nimble-select`, and `nimble-combobox`
```

Comment:
Think I'd prefer an `appearance-` prefix to align with `appearance` / `appearance-variant`.
I think just keep the name describing the state (it's a readonly visual that today only does anything when the control is in the disabled state, but I don't think that part needs to be encoded in the name).

So I'd go for `appearance-readonly` with the docs describing that it only applies when disabled.

Alternatives:
- `appearance-readonly` (my pref)
- `appearance-force-readonly`
- `appearance-override-readonly`
- `appearance-show-readonly`
- `appearance-as-readonly`
- `appearance-readonly-override`
- `appearance-readonly-force`